cranelift/assembler-x64/meta/src/generate/inst.rs

```diff
@@ -283,6 +288,26 @@ impl dsl::Inst {
             },
         );
     }
+
+    fn display_evex(f: &mut Formatter, inst: &dsl::Inst) {
+        for op in inst.format.operands.iter() {
+            let location = op.location;
+            let to_string = location.generate_to_string(op.extension);
+            fmtln!(f, "let {location} = {to_string};");
+        }
+        let ordered_ops = inst.format.generate_att_evex_style_operands();
+        let mut implicit_ops = inst.format.generate_implicit_operands();
+        if inst.has_trap {
+            fmtln!(f, "let trap = self.trap;");
+            if implicit_ops.is_empty() {
+                implicit_ops.push_str(" ;; {trap}");
+            } else {
+                implicit_ops.push_str(", {trap}");
+            }
+        }
+        fmtln!(f, "write!(f, \"{{name}} {ordered_ops}{implicit_ops}\")");
+    }
+
```

Comment:
If we ignore the opmask registers and masking behavior then we don't need this, right? Or I guess we still need to print `{k0}` or something like that to match Capstone? Maybe we should implement the "opmask registers as operands" logic after all...